content/zh/about/solutions/_index.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: 解决方案
 description: 解决方案。
-subtitle: Istio 目的是解决当单体应用程序向分布式微服务架构过渡时开发人员和运营商所面临的挑战。
+subtitle: Istio 目的是解决当单体应用程序向分布式微服务架构过渡时开发人员和运维人员所面临的挑战。
```

Comment:
The latest en version subtitle is:
``
subtitle: Learn how to succeed at security, observability, and traffic management initiatives using Istio.
``

Let's sync to the latest.